src/Form/OptionsApi.php

```diff
@@ -234,7 +229,7 @@ public function toArray(): array
      */
     public function url(): string
     {
-        return Str::template($this->url, $this->data());
+        return Str::safeTemplate($this->url, $this->data());
```

Comment:
@distantnative As far as I can tell, the `url()` method is only ever called from the `options()` method and there only ever for the URL or file to request the options from. So the URL doesn't seem to end up in the HTML, which means it probably (?) shouldn't be escaped for the HTML context. Maybe I'm missing something here.

Reply:
True, I think this was a too greedy Cmd+F for `Str::template`